.github/workflows/review.yaml

```diff
@@ -0,0 +1,31 @@
+name: Code Review
+
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+    branches: 
+      - main
+      - staging
+
+concurrency:
+  group:
+    ${{ github.repository }}-${{ github.event.number || github.head_ref ||
+    github.sha }}-${{ github.workflow }}-${{ github.event_name ==
+    'pull_request_review_comment' && 'pr_comment' || 'pr' }}
+  cancel-in-progress: ${{ github.event_name != 'pull_request_review_comment' }}
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: coderabbitai/ai-pr-reviewer@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GT_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        with:
+          debug: false
+          review_simple_changes: false
+          review_comment_lgtm: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The workflow file looks good overall, but there are a few improvements that could be made.

Firstly, it's generally a good practice to pin the version of the GitHub action you're using. Using `@latest` might lead to unexpected behavior if the action is updated with breaking changes. 

Secondly, the `GITHUB_TOKEN` environment variable seems to be sourced from a secret named `GT_TOKEN`. However, GitHub provides a built-in secret for this purpose, `secrets.GITHUB_TOKEN`, which has the necessary permissions and is automatically rotated.

Lastly, the file lacks a newline at the end. While this isn't a critical issue, it's a common convention to end files with a newline, as some tools might not handle files without a final newline correctly.

Here are the suggested changes:

```diff
    runs-on: ubuntu-latest
    steps:
      - uses: coderabbitai/ai-pr-reviewer@latest
+       - uses: coderabbitai/ai-pr-reviewer@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GT_TOKEN }}
+           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
        with:
          debug: false
          review_simple_changes: false
          review_comment_lgtm: false
\ No newline at end of file
+           review_comment_lgtm: false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->